### DISCUSSION TITLE
Latex Suite - Templater integration

### DISCUSSION THREAD

**Discussion by kenNash** [artisticat1/obsidian-latex-suite#230] [Ideas]
First of all, thanks for an amazing plugin!

I'm wandering if is it possible to integrate Latex Suite with Templater… it would be really great! ;-)

**Comment by artisticat1** [comment]
I don't use templates, so would you mind elaborating a bit more? Feel free to submit a feature request under Issues.

**Reply by kenNash** [reply]
As an example,
```
{ trigger: "td", replacement: "<% tp.date.now(\"YYYY-MM-DD\") %>", options: "tA"},
```
would print today's date, using one of the many possibilities of [Templater plugin](https://github.com/SilentVoid13/Templater).

**Comment by superle3** [comment]
the main default snippets will stay latex focused, but a pr adding a seperate file or repo having templater snippets is welcome

(I don't use templater so I won't really be able to create good snippets and help from outside is needed from templater users)
